daffodil-cli/src/main/scala/org/apache/daffodil/Main.scala

```diff
@@ -1584,35 +1584,35 @@ object Main extends Logging {
     } catch {
       case s: scala.util.control.ControlThrowable => throw s
       case e: java.io.FileNotFoundException => {
-        log(LogLevel.Error, "%s", e.getMessage())
+        log(LogLevel.Error, "Missing File Error", e.getMessage())
         1
       }
       case e: InvalidParserException => {
-        log(LogLevel.Error, "%s", e.getMessage())
-        1
+        log(LogLevel.Error, "Invalid Parser Error", e.getMessage())
+        2
```

Comment:
I would also like to see all the error codes defined as symbolic constants in one place. Eg., 

def INVALID_PARSER_ERROR = 2

with all of them in a block together so you can see them all.

 (btw. def vs. val we use def for constants.)